webcompat/static/css/development/components/form.css

```diff
@@ -49,10 +50,29 @@
   /* helpMessage */
   .wc-Form-helpMessage {
     font-weight: 700;
-    color: #ff0000;
+    color: var(--Form-helpMessageColor);
     margin: 0 0 0 .5em;
   }
 
+    .wc-Form-helpMessage--imageUpload {
+      position: absolute;
+      width: 100%;
+      bottom: 0;
+      left: 0;
+      right: 0;
+      padding: 1em 0;
+      font-weight:700;
+      background-color: var(--Form-helpMessageColor);
+      border-bottom-left-radius: 20px;
+      border-bottom-right-radius: 20px;
+      color: #fff;
+      cursor: pointer;
+      text-align: center;
+      text-decoration: underline;
+      transform: translateY(0);
```

Comment:
Hm, this is possibly copy-pasta that should be removed. I'm not using a `is-hidden` class to transition from.